.ci_support/linux_mpimpichpython2.7.yaml

```diff
@@ -9,7 +9,7 @@ channel_targets:
 docker_image:
 - condaforge/linux-anvil-comp7
 hdf5:
-- 1.10.4
+- 1.10.5
```

Comment:
This version of `h5py` does not work with `hdf5 >1.10.4`. For a version that does work [you should use the dev release](https://github.com/conda-forge/h5py-feedstock/tree/dev) or wait for upstream to issue a new release.

Reply:
Thanks @ocefpaf ... thats too bad.

Reply:
But on archlinux they managed see [here](https://git.archlinux.org/svntogit/community.git/commit/trunk?h=packages/python-h5py&id=d1d4f225b9315ff4ed40f2be7bd6881e8a4d0197).

Reply:
You can port that patch. I went for a dev release b/c I needed more than just hdf5-1.10.5 support.

Reply:
That is the patch that they are using: https://github.com/h5py/h5py/commit/141eafa531c6c09a06efe6a694251a1eea84908d.patch

Reply:
That is only skipping a failing test if I got it right.
I remember the devs mentioning other issues, not sure if that is enough.